src/main/java/net/fabricmc/loader/entrypoint/minecraft/hooks/EntrypointUtils.java

```diff
@@ -51,9 +51,8 @@
 			} catch (Throwable t) {
 				if (exception == null) {
 					exception = new RuntimeException("Could not execute entrypoint stage '" + name + "' due to errors, provided by '" + container.getProvider().getMetadata().getId() + "'!");
```

Comment:
Wouldn't the below fix the entire issue since specifying the cause on the RuntimeException would initialize this?

```suggestion
					exception = new RuntimeException("Could not execute entrypoint stage '" + name + "' due to errors, provided by '" + container.getProvider().getMetadata().getId() + "'!", t);
```

Reply:
oh nice right